waspc/cli/Command/Watch.hs

```diff
@@ -35,19 +44,45 @@ watch waspProjectDir outDir = FSN.withManager $ \mgr -> do
   chan <- newChan
   _ <- FSN.watchDirChan mgr (SP.toFilePath waspProjectDir) eventFilter chan
   _ <- FSN.watchTreeChan mgr (SP.toFilePath $ waspProjectDir </> Common.extCodeDirInWaspProjectDir) eventFilter chan
-  listenForEvents chan currentTime
+  timeOfLastEvent <- MVar.newMVar currentTime
+  listenForEvents chan currentTime timeOfLastEvent
   where
-    listenForEvents :: Chan FSN.Event -> UTCTime -> IO ()
-    listenForEvents chan lastCompileTime = do
+    oneSecond :: Int
```

Comment:
This is micro seconds right? In that case let's name it `oneSecondInMicroSeconds`.